cmd/lock-rest-server.go

```diff
@@ -247,10 +274,13 @@ func startLockMaintenance() {
 		case <-ticker.C:
 			// Start with random sleep time, so as to avoid
 			// "synchronous checks" between servers
-			r := rand.New(rand.NewSource(UTCNow().UnixNano()))
 			duration := time.Duration(r.Float64() * float64(lockMaintenanceInterval))
 			time.Sleep(duration)
-			lockMaintenance(lockValidityCheckInterval)
+			if err := lockMaintenance(ctx, lockValidityCheckInterval, objAPI); err != nil {
```

Comment:
Shouldn't the go-routine sleep for a random duration before calling lock maintenance? With this PR we could be calling lock maintenance synchronously across MinIO servers increasing unproductive contention.

Reply:
yes, it's already sleeping right before the call @krisis r.Float64() returns random value every time its called.  Also because we are holding global lock only one will run at any given point in time.